ext/OceananigansNCDatasetsExt.jl

```diff
@@ -600,9 +603,9 @@ default_tracer_attributes(::BuoyancyForce{<:SeawaterBuoyancy{FT, <:LinearEquatio
     "T" => Dict("long_name" => "Temperature", "units" => "°C"),
     "S" => Dict("long_name" => "Salinity",    "units" => "practical salinity unit (psu)"))
 
-# Assuming TEOS-10 I guess?
-default_tracer_attributes(::SeawaterBuoyancy) = Dict("T" => Dict("long_name" => "Conservative temperature", "units" => "°C"),
-                                                     "S" => Dict("long_name" => "Absolute salinity",        "units" => "g/kg"))
+default_tracer_attributes(::BuoyancyBoussinesqEOSModel) = Dict(
+        "T" => Dict("long_name" => "Conservative temperature", "units" => "°C"),
+        "S" => Dict("long_name" => "Absolute salinity",        "units" => "g/kg"))
```

Comment:
```suggestion
default_tracer_attributes(::BuoyancyBoussinesqEOSModel) = Dict("T" => Dict("long_name" => "Conservative temperature", "units" => "°C"),
                                                               "S" => Dict("long_name" => "Absolute salinity",        "units" => "g/kg"))
```

I think this follows closer to the Oceananigans convention.